acorn/src/options.js

```diff
@@ -91,14 +91,25 @@ export const defaultOptions = {
 
 // Interpret and default an options object
 
+let warnedAboutEcmaVersion = false
+
 export function getOptions(opts) {
   let options = {}
 
   for (let opt in defaultOptions)
     options[opt] = opts && has(opts, opt) ? opts[opt] : defaultOptions[opt]
 
-  if (options.ecmaVersion >= 2015)
+  if (options.ecmaVersion === "latest") {
+    options.ecmaVersion = 1e8
+  } else if (options.ecmaVersion == null) {
+    if (!warnedAboutEcmaVersion && typeof console === "object" && console.warn) {
+      warnedAboutEcmaVersion = true
+      console.warn("Since Acorn 8.0.0, options.ecmaVersion is required.\nDefaulting to 2020, but this will stop working in the future.")
+    }
+    options.ecmaVersion = 11
+  } else if (options.ecmaVersion >= 2015) {
```

Comment:
I wonder if, as part of this version overhaul, we should deprecate non-year based versions beyond ES6 (ES7?). I don't think anyone uses ES8+ in the wild, and it might be good to enforce a single standard that community prefers.

Reply:
It costs us almost nothing to support these, and since we'll need to support non-year versions for the old versions anyway, it seems reasonable to support them for all versions.

Reply:
It costs nothing, but it creates a weird split IMO. I'm a big fan of consistency :D